### PR TITLE
Disable typing datetime. Fix display errors when start time or finish…

### PR DESCRIPTION
**Pull request by ducnt-0269**
… time is empty, and validate start finish time

### CODE REVIEW

File: app/assets/javascripts/kendo.custom.js [hunk 1]
```diff
@@ -88,4 +88,7 @@ $(document).ready(function() {
 
   schedule_start_time.max(schedule_finish_time.value());
   schedule_finish_time.min(schedule_start_time.value());
+  
+  $("#schedule_start_time").attr("readonly","readonly");
```
Comment (dieunb) [inline app/assets/javascripts/kendo.custom.js]: missing a space after  `attr("readonly",`

File: app/assets/javascripts/kendo.custom.js [hunk 2]
```diff
@@ -88,4 +88,7 @@ $(document).ready(function() {
...
+  $("#schedule_finish_time").attr("readonly","readonly");
```
Comment (dieunb) [inline app/assets/javascripts/kendo.custom.js]: missing a space after  `attr("readonly",`

**Comment by vanlinha3k35pbc** [comment]
OK

File: app/models/schedule.rb [hunk 3]
```diff
@@ -68,20 +68,22 @@ def valid_room
   end
 
   def valid_time
-    if !start_time.blank? && !finish_time.blank? && start_time >= finish_time
-      errors.add :start_time, I18n.t("invalid_start_time")
-    end
+    unless start_time.blank? || finish_time.blank?
```
Comment (dieunb) [inline app/models/schedule.rb:71]: if start time is blank or finish time is blank, what is error message?

Comment (ducnt-0269) [inline app/models/schedule.rb:71]: validates :start_time, presence: true
validates :finish_time, presence: true
Will take care of it

**Pull request merged**